### DISCUSSION TITLE
배포와 롤백 강의

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#74] [General]
# 배포와 롤백 강의

무중단 배포 서비스를 다운시키지 않고 배포하는 것. 요즘 큰 회사들은 서비스 점검 공지를 보이지 않는다. 인프라 변경 JPA DDL auto 설정 또는 flyway 를 이용하여 운영 DB의 스키마를 관리하고 있지는 않은가? spring.jpa.hib…

https://vero.wiki/rollback-release/

**Comment by carsago** [comment]
베로 캠에서 와리가리 하는거 땜에 DM하다가 참았었는데

**Reply by Cyma-s** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋ아 다 보이네

**Comment by carsago** [comment]
DM할려다가, 아니 이거 댓글 수정 왜 없지
